examples/EdgeCaseSaliency.ipynb

```diff
@@ -0,0 +1,1318 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "47395fb4",
+   "metadata": {},
+   "source": [
+    "# Saliency to Explain Edge Cases\n",
+    "\n",
+    "This demo demonstrates the use of the `xaitk_saliency` package to provide useful explanations for the performance of an object detection model.\n",
+    "Specifically, we examine instances where the decisions the model makes are questionable.\n",
+    "Saliency maps can provide useful insights into why the model behaved in an unexpected way."
```

Comment:
We should more specifically introduce the use-case right off the bat. This by itself reads a too generic to me and doesn't really give me an idea of what the rest of the notebook/demo is going to entail. We should treat this first cell like a paper abstract and introduce a complete sketch of what will follow.